### DISCUSSION TITLE
How can I open current dir?

### DISCUSSION THREAD

**Discussion by adriangalilea** [sxyazi/yazi#1392] [1. Q&A] [answered]
```
[[input.prepend_keymap]]
	on   = "e"
	run  = "open"
	desc = "open dir"
```

I want to open the current path with helix, but I'm not sure how to do it.

**Comment by lpnh** [comment, marked as answer]
See if the `o` key works.

**Reply by adriangalilea** [reply]
That was it, thanks.

**Comment by sxyazi** [comment]
`run = "shell --block --confirm 'hx .'"` should work

Doc of `shell` command: https://yazi-rs.github.io/docs/configuration/keymap#manager.shell
